lib/utils/autocomplete.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const homedir = require('os').homedir();
+const readFileSync = require('./fs/readFileSync');
+const fileExistsSync = require('./fs/fileExistsSync');
+
+const tab = require('tabtab')({
+  name: 'serverless',
+});
+
+const autocomplete = () => {
+  const cacheFilePath = `${homedir}/.serverless/autocomplete.json`;
```

Comment:
This cache file path doesn't take in to account multiple projects which could have different autocomplete values since they have different plugins. I would create a cache file on a per project basis based off a hash of the path.

Something to this effect...
```js
const cacheFilePath = `${homedir}/.serverless/${md5(projectPath)}/autocomplete.json`;
```